toolchain/check/convert.cpp

```diff
@@ -383,12 +384,15 @@ static auto ConvertTupleToTuple(Context& context, SemIR::TupleType src_type,
                                                      new_block.id()});
 }
 
-// Performs a conversion from a struct to a struct type. Does not perform a
-// final conversion to the requested expression category.
-static auto ConvertStructToStruct(Context& context, SemIR::StructType src_type,
-                                  SemIR::StructType dest_type,
-                                  SemIR::NodeId value_id,
-                                  ConversionTarget target) -> SemIR::NodeId {
+// Performs a conversion from a struct to a struct or class type. Does not
+// perform a final conversion to the requested expression category.
```

Comment:
Instead of "Does not perform", maybe "The caller should perform ... if necessary"? Or maybe a TODO if that's the intent? (the latter is where I'm a little uncertain from phrasing; similar elsewhere you have this comment)

Reply:
I'd prefer to state a fact here rather than give advice. Would changing "Does not" to "Intentionally does not" help explain that this is not a TODO?

Reply:
Added a few more words to make this sound less TODO-y.

Reply:
Yes, thanks!